test/OrchardCore.Tests/Extensions/ExtensionManagerTests.cs

```diff
@@ -25,13 +27,17 @@ public class ExtensionManagerTests
         private readonly ExtensionManager _themeScopedExtensionManager;
         private readonly ExtensionManager _moduleThemeScopedExtensionManager;
 
+#pragma warning disable CA1859 // Use concrete types when possible for improved performance - not relevant for unit tests.
+        private readonly ITypeFeatureProvider _moduleScopedTypeFeatureProvider = new TypeFeatureProvider();
+#pragma warning restore CA1859 // Use concrete types when possible for improved performance
```

Comment:
While it's not a matter of measurable performance difference, you can still use the concrete type without drawbacks, and the code is nicer than with an otherwise unnecessary interaction with the analyzer.

```suggestion
        private readonly TypeFeatureProvider _moduleScopedTypeFeatureProvider = new TypeFeatureProvider();
```

Reply:
Yeah, I thought about that, but I explicitly didn't want to tie the unit test to the specific type. The type could be a mock in future too.

Reply:
Then we can change this in the future :).